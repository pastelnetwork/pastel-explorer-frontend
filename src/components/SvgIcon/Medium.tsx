const Medium = () => {
  return (
    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
      <path d="M1.43998 2.88086L3.35998 5.32492V16.9377L0.47998 20.6409H7.19998L4.31998 16.9377V6.60086L10.56 20.6409L10.559 20.6474L16.32 6.50242V18.7209L14.4 20.6409H22.56L20.64 18.7209L20.6269 5.04273L22.4944 2.88742H16.7259L12.2906 13.798L7.43811 2.88086H1.43998Z" />
    </svg>
  );
};

export default Medium;
