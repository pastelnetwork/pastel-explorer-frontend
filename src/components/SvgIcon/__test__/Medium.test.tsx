import { shallow } from 'enzyme';

import Medium from '../Medium';

describe('components/SvgIcon/Medium', () => {
  const wrapper = shallow(<Medium />);

  test('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should render <svg>', () => {
    expect(wrapper.find('svg').length).toBe(1);
  });

  test('should render <path>', () => {
    expect(wrapper.find('path').length).toBe(1);
  });
});
