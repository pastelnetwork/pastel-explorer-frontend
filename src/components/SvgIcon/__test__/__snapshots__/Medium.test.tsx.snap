// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/SvgIcon/Medium renders correctly 1`] = `ShallowWrapper {}`;
